vue-components/src/components/Lookup.vue

```diff
@@ -145,7 +199,7 @@ export default Vue.extend( {
 		},
 
 		showsMenu(): boolean {
-			return !this.hasItemSelected && this.focused && this.searchInput.length > 0;
+			return !this.escKeyPressed && !this.enterKeyPressed && this.focused && this.searchInput.length > 0;
```

Comment:
After some initial skepticism, I tried it out and I like it! Have a look at the current implementation.

I could not yet get rid of `this.enterKeyPressed` as the blur behavior in `onSelect` differs depending on whether the item was selected with Enter or click. This might need some more thinking.

Reply:
Hmm, I would question the reasoning behind the different modes of behaviour, in all other implementations of an autocomplete I know there's no difference between enter and mouse click. Furthermore, the fact that the element blurs after mouse click interrupts a flow where a user uses both keyboard and mouse navigation as the visual focus signifier is lost on click. @SaiSan-WMDE what do you think?

Reply:
Hi! I would encourage us to make the component behave the same way whenever a menu item is selected either on mouse down or enter: the selection is made and the focus remains on the input element.

Reply:
Mh, I encoded that in https://phabricator.wikimedia.org/T265884#6584975 and I did it because I read it somewhere, but I can't find it in Figma now o.O

I'll change it and then we can drop `this.enterKeyPressed` completely :raised_hands: